Godunov/hydro_godunov_extrap_vel_to_faces_3D.cpp

```diff
@@ -430,7 +432,7 @@ Godunov::ExtrapVelToFacesOnBox ( Box const& bx, int ncomp,
             sth += Real(0.5) * l_dt * f(i  ,j,k,n);
         }
 
-        HydroBC::SetXEdgeBCs(i, j, k, n, q, stl, sth, bc.lo(0), dlo.x, bc.hi(0), dhi.x, true);
+        HydroBC::SetXEdgeBCs(i, j, k, n, q, stl, sth, stl, sth, bc.lo(0), dlo.x, bc.hi(0), dhi.x, true);
```

Comment:
Are these calls a substitute for `Godunov_cc_zbc_lo` type calls in A-W? In A-W, those use `u_ad/v_ad/w_ad` as the "edge" velocities. Are you sure using `stl/sth` is correct?

Reply:
Yes, on a second look, to match with A-W, these calls should be passing `u_ad/v_ad/w_ad` instead of `stl` and `sth` as `velm` and `velp`.

See the `Godunov_cc_xbc_lo` type calls in `incflo_godunov_predict.cpp` in A-W for reference. Note that these are coded separately from the `Godunov_trans_xbc` type functions in A-W. They are mostly similar in implementation though but need to make sure..